src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) =>
+    century
+      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
+      : person.sex === 'm');
+  const menAges = men.map((person) => person.died - person.born);
+  const sumOfMenAges = menAges.reduce((a, b) => a + b);
+
+  return sumOfMenAges / menAges.length;
```

Comment:
also for this part and reduce you can create help function passing an array and return `yourFunc(men)`